dotcom-rendering/src/client/index.apps.ts

```diff
@@ -0,0 +1,36 @@
+import './webpackPublicPath';
+
+// these modules are bundled in the initial (i.e. this) chunk, so that they run ASAP
+import { dynamicImport } from './dynamicImport';
+import { ga } from './ga';
+import { islands } from './islands';
+import { recordInitialPageEvents } from './ophan/recordInitialPageEvents';
+import { performanceMonitoring } from './performanceMonitoring';
+import { sentryLoader } from './sentryLoader';
+import { startup } from './startup';
+
+startup('recordInitialPageEvents', recordInitialPageEvents);
+startup('ga', ga);
+startup('sentryLoader', sentryLoader);
+startup('dynamicImport', dynamicImport);
+startup('islands', islands);
+startup('performanceMonitoring', performanceMonitoring);
+
+// these modules are loaded as separate chunks, so that they can be lazy-loaded
```

Comment:
would we want to lazy-load these do you think? My instinct is that to support offline reading, we want everything in the same bundle - but maybe our build does that anyway!